README.md

```diff
@@ -40,75 +39,42 @@ The installation procedure has been tested on Ubuntu 20.04 and Windows 10.
 On **Linux** or **macOS**, run:
 
 ```bash
+mamba activate  <conda-environment-name>
```

Comment:
I am a bit confused on this. I thought the current supported workflows in the ironcub team were:
* W1: Linux: dependencies installed with apt + robotology-superbuild from source (for YARP, idyntree, ..) + ironcub software via colcon
* W2: Windows: dependencies installed via mamba from conda-forge and robotology channel + ironcub software via colcon

I do not know if:

* W3: Linux: dependencies installed via mamba from conda-forge and robotology channel + ironcub software via colcon

was actually supported (i.e. used by someone).

So I tought that for this repo, the workflow were similar, just substituting "ironcub software via colcon" with "ironcub_mk1_software installed via cmake". However, this modifications are breaking the workflow W1. If that is intentional, probably we also need to modify the top of the readme were we suggest users to install deps via the robotology-superbuild .

Reply:
>W1: Linux: dependencies installed with apt + robotology-superbuild from source (for YARP, idyntree, ..) + ironcub software via colcon

I am using this workflow on Ubuntu, I never tested W3 on my pc for this repo and I don't know if someone did it.

Partially related, I think it would be better to at least specify a tag for the robotology superbuild and the other dependencies. For now I have recently tested this repo with the latest robotology-superbuild and worked so we can use the most recent tags.

Reply:
My bad! I was thinking of dropping the possibility of configuring this repository via source. I have updated the documentation accordingly.
@traversaro @gabrielenava could you please review it again? Thanks :)